per-directory-history.zsh

```diff
@@ -123,9 +123,9 @@ function _per-directory-history-addhistory() {
          [[ -o inc_append_history ]] || \
          [[ -o inc_append_history_time ]]; then
           fc -AI $HISTFILE
```

Comment:
Do we need to quote here as well?

Reply:
I had overlooked them, thanks. Fixed. There are still a few variables that are not quoted, but I can assume they are safe.